.github/workflows/build-and-test.yml

```diff
@@ -192,16 +192,8 @@ jobs:
             /home/runner/go/pkg/mod
             /home/runner/.cache/go-build
           key: go-pkg-mod-${{ runner.os }}-${{ hashFiles('**/go.mod', '**/go.sum') }}
-      - name: Cache Tools
```

Comment:
Why removed?

Reply:
Not needed.

Reply:
mdatagen is used by build but probably not worth the cost of restoring cache just for that.